deployment/terraform/cloud-config/ecs-batch-user-data.yml

```diff
@@ -0,0 +1,78 @@
+#cloud-config
+
+packages:
+    - awslogs
+
+fs_setup:
```

Comment:
Above when you mention that the block device mapping isn't needed, is also true that we can omit this stuff too?